ocaml-lsp-server/src/dune.ml

```diff
@@ -28,6 +28,7 @@ module Chan = struct
           match Csexp.input_opt t.in_chan with
           | Error _
           | Ok None ->
+            Unix.close (Unix.descr_of_in_channel t.in_chan);
```

Comment:
Ideally, we could log this and send a notification to the client

Reply:
Yeah, that makes sense. Note that this case always corresponds to a bug in `dune rpc init`. It should never crash or terminate on its own.